File: duplicate_elements/duplicate.c
#include <stdio.h>

int containsDuplicate(int *nums, int numsSize)
{
    int i, j;
    for (i = 0; i < numsSize; i++)
    {
        for (j = i + 1; j < numsSize; j++)
        {
            if (nums[i] == nums[j])
            {
                return 1;
            }
        }
    }
    return 0;
}

void main()
{
    int li[6] = {1, 2, 3, 4, 5, 1};
    int res = containsDuplicate(li, 6);
    if (res)
    {
        printf("the list have duplicate");
    }
    else
    {
        printf("the list dosent have duplicate");
    }
}

File: duplicate_elements/duplicate.py
def find_dup(li):
    return len(li) != len(set(li))


li = [1, 2, 3, 4, 1]
res = find_dup(li)
if res:
    print("the list have duplicate elemetnts")
else:
    print("the list dosent have duplicate elements")
